lib/membrane/rtp/inbound_packet_tracker.ex

```diff
@@ -74,35 +74,29 @@ defmodule Membrane.RTP.InboundPacketTracker do
   end
 
   @impl true
-  def handle_process(:input, buffer, _ctx, %State{cycles: cycles, max_seq: max_seq} = state) do
+  def handle_process(:input, buffer, _ctx, %State{} = state) do
     seq_num = buffer.metadata.rtp.sequence_number
-    max_seq = max_seq || seq_num - 1
 
-    delta = rem(seq_num - max_seq + @max_seq_num + 1, @max_seq_num + 1)
+    {delta, packet_index, tracker} = SequenceNumberTracker.track(state.seq_num_tracker, seq_num)
 
-    cond do
-      # greater sequence number but within dropout to ensure that it is not from previous cycle
-      delta < @max_dropout ->
-        state =
-          state
-          |> update_sequence_counters(seq_num, max_seq, cycles)
-          |> update_received()
-          |> update_jitter(buffer)
+    state =
+      %State{state | base_seq: state.base_seq || packet_index, seq_num_tracker: tracker}
+      |> update_received()
 
-        {[buffer: {:output, repair_sequence_number(buffer, state)}], state}
+    lost_ids = Enum.to_list((packet_index - (delta - 1))..(packet_index - 1)//1)
 
-      # the packets is either too old or too new
-      delta <= @max_seq_num - @max_unordered ->
-        {[], update_received(state)}
+    if abs(delta) > @max_unordered do
+      # Ignore packet
+      {[], state}
```

Comment:
I'm not 100% sure, I've just combined the existing conditions. The previous version dropped packets with diff >= 3000 and < (65535 - 3000). I believe this is to rule out a scenario where we receive a terribly late packet and consider it a newer one, breaking the sequence number tracking and causing an enormous amount of NACKs to be sent

The scenario example:
```elixir
test "huge gap" do
  tracker = Tracker.new()
  assert {1, 1, tracker} = Tracker.track(tracker, 1)
  assert {32_767, 32_768, tracker} = Tracker.track(tracker, 32_768)
  assert {-32_766, 2, tracker} = Tracker.track(tracker, 2)
  assert {-32_765, 3, tracker} = Tracker.track(tracker, 3)
end
```

However, as shown above, this will break the `SequenceNumberTracker` anyway.

Reply:
Now state update is only done when we're not dropping packets and increased the limit 3x.
I've also added some comments